application/modules/invoices/models/Mdl_invoices.php

```diff
@@ -31,7 +31,7 @@ public function statuses()
                 'href' => 'invoices/status/draft'
             ),
             '2' => array(
-                'label' => trans('sent'),
+                'label' => trans('Sent'),
```

Comment:
The translation string is 'sent', not 'Sent'. It's being looked up in `ip_lang.php`. The old translation string was correct

Reply:
I made them all lowercased